deploy/packer/base/scripts/packages.sh

```diff
@@ -3,7 +3,7 @@
 set -e -v
 
 DOCKER_VERSION=17.12.0~ce-0~ubuntu
-KERNEL_VER=4.4.0-109-generic
+KERNEL_VER=4.4.0-1014-aws
```

Comment:
Shouldn't this have `taskcluster` in its version to match the custom kernel version?

Reply:
I don't fully understand the apt matching algorithm, but the short answer is *no*. `apt-get` will manage to match the correct version, I checked that.